test/java/org/chromium/net/CronetUrlRequestTest.java

```diff
@@ -513,7 +516,7 @@ public void testAddHeader() throws Exception {
   @Test
   @SmallTest
   @Feature({"Cronet"})
-  @Ignore("MockWebServer does not handle header field with multiple values like NativeTestServer")
+  @Ignore("Debug me")
```

Comment:
Is this broken?

Reply:
There are two tests for which I don't have a good explanation yet. I will take care of them later.

Reply:
Sounds good!